test/funnels/Depositor.t.sol

```diff
@@ -0,0 +1,595 @@
+        deal(DAI,  address(this), 1_000_000 * WAD,   true);
+        GemLike(DAI).approve(UNIV3_ROUTER, 1_000_000 * WAD);
+        SwapRouterLike.ExactInputParams memory params = SwapRouterLike.ExactInputParams({
+            path:             DAI_USDC_PATH,
+            recipient:        address(this),
+            deadline:         block.timestamp,
+            amountIn:         1_000_000 * WAD,
+            amountOutMinimum: 990_000 * 10**6
+        });
+        SwapRouterLike(UNIV3_ROUTER).exactInput(params);
+
+        dp.liquidity = liq;
+        vm.warp(block.timestamp + 3600);
+        vm.expectEmit(true, true, true, false);
+        emit Withdraw(FACILITATOR, DAI, USDC, liq, 0, 0, 0, 0);
+        vm.prank(FACILITATOR); (, uint256 withdrawn0, uint256 withdrawn1) = depositor.withdraw(dp, true);
+
+        assertTrue(
+            (withdrawn0 > deposited0 && GemLike(DAI ).balanceOf(address(buffer)) > initialDAI ) ||
+            (withdrawn1 > deposited1 && GemLike(USDC).balanceOf(address(buffer)) > initialUSDC)
+        );
+        assertTrue(GemLike(DAI).balanceOf(address(buffer)) >= prevDAI);
+        assertTrue(GemLike(USDC).balanceOf(address(buffer)) >= prevUSDC);
+        assertEq(GemLike(DAI).balanceOf(address(depositor)), 0);
+        assertEq(GemLike(USDC).balanceOf(address(depositor)), 0);
+        assertEq(_getLiquidity(DAI, USDC, 100, REF_TICK-100, REF_TICK+100), 0);
+    }
+
+    function testWithdrawAmounts() public {
+        Depositor.LiquidityParams memory dp = Depositor.LiquidityParams({
+            gem0: DAI,
+            gem1: USDC,
+            fee: uint24(100),
+            tickLower: REF_TICK-100,
+            tickUpper: REF_TICK+100,
+            liquidity: 0,
+            amt0Desired: 500 * WAD,
+            amt1Desired: 500 * 10**6,
+            amt0Min: 490 * WAD,
+            amt1Min: 490 * 10**6
+        });
+        vm.prank(FACILITATOR); (uint128 liq, uint256 deposited0, uint256 deposited1) = depositor.deposit(dp);
+        assertGt(_getLiquidity(DAI, USDC, 100, REF_TICK-100, REF_TICK+100), 0);
+
+        dp.liquidity = 0;
+        dp.amt0Desired = deposited0;
+        dp.amt1Desired = deposited1;
+
+        uint256 liquidityBeforeWithdraw = _getLiquidity(DAI, USDC, 100, REF_TICK-100, REF_TICK+100);
+
+        vm.warp(block.timestamp + 3600);
+        vm.expectEmit(true, true, true, false);
+        emit Withdraw(FACILITATOR, DAI, USDC, liq, 0, 0, 0, 0);
+        vm.prank(FACILITATOR); (, uint256 withdrawn0, uint256 withdrawn1) = depositor.withdraw(dp, false);
+
+        // due to liquidity from amounts calculation there is rounding dust
+        assertTrue(withdrawn0 * 100001 / 100000 >= deposited0);
+        assertTrue(withdrawn1 * 100001 / 100000 >= deposited1);
+        assertEq(GemLike(DAI).balanceOf(address(depositor)), 0);
+        assertEq(GemLike(USDC).balanceOf(address(depositor)), 0);
+        assertLe(_getLiquidity(DAI, USDC, 100, REF_TICK-100, REF_TICK+100), liquidityBeforeWithdraw);
```

Comment:
I think it should be "assertLt"

Reply:
Changed in [f23d585](https://github.com/makerdao/dss-allocator/pull/2/commits/f23d585b25acdf5f855375d4d67f5ec37a07aee0)